config/rootrc.in

```diff
@@ -242,6 +242,7 @@ GeomPainter.Name:            @root_geompainter_type@
 # THttpServer specific settings
 # location of JavaScript ROOT sources
 #HttpServ.JSRootPath:        @jsrootdir@
+JSROOT.Enable:               on
```

Comment:
And here add small comments that it means at how `%jsroot on/off` can be used in jupyter to change this